src/main/java/org/opensearch/eval/App.java

```diff
@@ -149,158 +149,4 @@ public static void main(String[] args) throws Exception {
 
     }
 
-//
```

Comment:
This is all old unused plugin code. The placeholder is in it on line 156 but none of the code is needed so removing rather than just changing the unused placeholder.